tests/integration/components/jukebox-onboarding-message/component-test.js

```diff
@@ -0,0 +1,28 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | onboarding-message', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`{{jukebox-onboarding-message}}`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      {{#jukebox-onboarding-message}}
```

Comment:
I don't think you need the block version of this test since the onboarding message component doesn't really get invoked by using a block, you're just using it as `{{jukebox-onboarding-message}}`, so you could delete lines 14-23